ip-messaging/rest/accounts/instance-get-example-1/meta.json

```diff
@@ -0,0 +1,4 @@
+{
+  "title": "Get account details",
```

Comment:
We should use title case in the snippet titles, I'll check all the existing ones in master to see if they are using this case. But let's fix this from now on